doc/reference/tooling/tt_cli/start.rst

```diff
@@ -52,19 +73,58 @@ process for status checks (:ref:`tt status <tt-status>`) and application stoppin
     If you start such an application with ``tt start``, ``tt`` won't be able to check
     the application status or stop it using the corresponding commands.
 
-Examples
---------
+.. _tt-start-integrity-check:
+
+Integrity check
+---------------
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    The integrity check functionality is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+``tt start`` can perform initial and periodical integrity checks of the environment,
+application, and centralized configuration.
+
+To enable the integrity checks of environment and application files, you need to pack
+the application using ``tt pack`` with the ``--with-integrity-check`` option.
+This option generates and signs checksums of executables and configuration files in the current ``tt``
+environment. Learn more in :ref:`tt-pack-integrity-check`.
+
+To enable the configuration integrity check, publish the configuration to a centralized
```

Comment:
The configuration file in general will be packed with the application. So the local configuration is exist.
```suggestion
To add the configuration at the centralized storage integrity check, publish the configuration to a centralized
```